org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/json/adapters/MessageTypeAdapter.java

```diff
@@ -42,6 +43,7 @@
 import com.google.gson.JsonSyntaxException;
 import com.google.gson.TypeAdapter;
 import com.google.gson.TypeAdapterFactory;
+import com.google.gson.internal.Primitives;
```

Comment:
@dhuebner Can you please have a look at https://github.com/eclipse-lsp4j/lsp4j/issues/738 in relation to this new package dependency?

Reply:
@jonahgraham 
Sure, will do. Thanks!